plugins/sigma/tests/tests.cpp

```diff
@@ -158,7 +158,25 @@ TEST(modifier - re) {
     foo|re: "^.*$"
   )__";
   auto search_id = to_search_id(yaml);
-  auto expected = to_expr("foo == /^.*$/");
+  auto expected = to_expr("foo == /^.*$/i");
+  CHECK_EQUAL(search_id, expected);
+}
+
+TEST(modifier - re 2) {
+  auto yaml = R"__(
+    foo|re: '.*foobar.*'
+  )__";
+  auto search_id = to_search_id(yaml);
+  auto expected = to_expr("foo == /.*foobar.*/i");
```

Comment:
Sanity check: is the `|re` modifier also implying case insensitivity in the spec?

Reply:
I took another look at the spec and am bit confused: it seems there is no way to produce a case-insensitive regex.

I've asked around and got the answer that "you can start your regex with `(?i)` if you want case insensitivity". I would argue that we don't do that until a user asks for it, though, as it's not spec-compliant.